### ISSUE TITLE
com.auth0.api.APIClient﹕ Failed obtain delegation token info

### ISSUE THREAD

**Issue by grestuccia1** [auth0/Lock.Android#117]
When try to generate new JWT with refresh_token

getLock().getAPIClient().fetchIdTokenWithRefreshToken(REFRESH_TOKEN), null, new RefreshIdTokenCallback() { ....

Gradle config
    compile 'com.auth0.android:lock:1.9.+'

**Comment by hzalaz** [comment]
@grestuccia1 try sending this
```java
Map<String, Object> parameters = ParameterBuilder.newEmptyBuilder().set("api_type", "app").asDictionary();
```
as the second parameter of the method `fetchIdTokenWithRefreshToken`

**Comment by hzalaz** [comment]
It's a known issue when you have an add-on configured in your Auth0 application (like AWS or Firebase). We will release a fix for this but that workaround should work.

**Comment by grestuccia1** [comment]
Thanks!

**Comment by hzalaz** [comment]
Closed in PR #121
